xbmc/guilib/GUITextLayout.h

```diff
@@ -187,7 +187,7 @@ class CGUITextLayout
   inline bool CanWrapAtLetter(character_t letter) const XBMC_FORCE_INLINE
   {
     character_t ch = letter & 0xffff;
-    return ch == L' ' || (ch >=0x4e00 && ch <= 0x9fff);
+    return ch == L' ';
```

Comment:
The previous condition that included the "CJK Unified Ideographs" block is strange and dates back to the original xbox code. No problem with the removal, though we won't know why it was added.
However maybe add a todo comment that unicode spaces are not handled (yet) - category Zs or maybe attribute white space. If that happens one day, non-breaking space will require special handling.

Reply:
i can add the comment, anyway here its not the only place where would be needed handle also unicode spaces

Reply:
done